src/test/java/TestPrinter.java

```diff
@@ -0,0 +1,21 @@
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.SystemOutRule;
+import training.supportbank.Printer;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class TestPrinter {
+    @Rule
+    public final SystemOutRule systemOutRule = new SystemOutRule().enableLog();
+
+    @Test
+    public void listAllPrintsToConsole() throws IOException {
+        Printer printer = new Printer();
+        String expectedOutput = "Jon A is owed £7.80\nStephen S is owed £4.37\nSarah T owes £7.80\nTim L owes £4.37\n";
+        printer.listAll("TestFile.csv");
+        assertThat(systemOutRule.getLog()).isEqualToIgnoringNewLines(expectedOutput);
```

Comment:
I didn't know you could do this. 👍

Reply:
Glad I could impart some knowledge to you! 😃